src/ploomber/spec/dagspec.py

```diff
@@ -565,7 +566,7 @@ def from_directory(cls, path_to_dir):
             The directory to use.  Looks for scripts
             (``.py``, ``.R`` or ``.ipynb``) in the directory and interprets
             them as task sources, file names are assigned as task names
-            (without extension). The spec is generated with the default values
+            (without extension). The spec is generated with the default valueq2 A2AAAAAAs
```

Comment:
looks like some debugging happened here haha

Reply:
Haha my keyboard has a mind of its own sometimes